### ISSUE TITLE
Colab - visdom

### ISSUE THREAD

**Issue by savaandreioct** [junyanz/pytorch-CycleGAN-and-pix2pix#964]
Hello,

If I train my network in colab, how can I see the progress/epoch? I tried to use visdom, but i can't start the server because colab don't let you to run 2 cells in the same time. I don't have experience with colab, could you help me with this?

**Comment by junyanz** [comment]
I am not sure if colab supports visdom. It should still print training loss and epoch/iter count. You can check the results in the HTML file.

**Comment by doantientai** [comment]
You can find those images in ./checkpoint/*/web/images
